src/test/java/org/fundacionjala/coding/manuel/Kata8Test.java

```diff
@@ -0,0 +1,155 @@
+package org.fundacionjala.coding.manuel;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+/**
+ * Created by Juan Manuel on 3/11/2018.
+ */
+public class Kata8Test {
+    private Kata8 kata8;
+    private static final String COMMON_LINE1 = " _  _  _  _  _  _  _  _  _ ";
+    private static final String COMMON_LINE2 = "|_||_||_||_||_||_||_||_||_|";
+    private static final String COMMON_LINE3 = "  |  |  |  |  |  |  |  |  |";
+    private static final String COMMON_LINE4 = " _| _| _| _| _| _| _| _| _|";
+    /**
+     * Method to initialize.
+     */
+    @Before
+    public void initialize() {
+        kata8 = new Kata8();
+    }
+
+    /**
+     * First test.
+     */
+    @Test
+    public void test1BankKata8() {
+        //" _  _  _  _  _  _  _  _  _ ";
+        String secondRow = "| || || || || || || || || |";
+        //"|_||_||_||_||_||_||_||_||_|";
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)